bin/index.js

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env node
+var ArgumentParser, argparser, argv, base58, buf, encodingOption, info,
```

Comment:
Does it not already? I compiled it with `cat bin/index.coffee | coffee -b -c -s | standard-format - > bin/index.js ` and checked it with the `standard` tool (which gave no errors).